sprocs/instance_questions_update_score.sql

```diff
@@ -163,7 +163,9 @@ BEGIN
             score_perc_in_grading = 0,
             status = 'complete',
             modified_at = now(),
-            highest_submission_score = new_score
+            highest_submission_score = new_score,
+            requires_manual_grading = FALSE,
+            last_grader = arg_authn_user_id
```

Comment:
Since this sproc is also used for score updates, does this mean we're treating score uploads as a form of manual grading? Intuitively this makes sense, but I do worry that setting `requires_manual_grading` and `last_grader` could be a little confusing for instructors... I'm not sure what I'd expect the behavior to be tbh.

Reply:
> Since this sproc is also used for score updates, does this mean we're treating score uploads as a form of manual grading? Intuitively this makes sense, but I do worry that setting `requires_manual_grading` and `last_grader` could be a little confusing for instructors... I'm not sure what I'd expect the behavior to be tbh.

Yes, that is my expectation, that (at least for now) any manual update of the score would reset the manual grading flag and update the last grader. I'm happy to add that in the docs once we start updating docs... I added a note in our scope document to ensure this is documented. While I can certainly entertain other options once this is more widely distributed, I believe this accounts for the common cases people use to update grades for manual grading questions without the manual grading interface.

Reply:
Makes sense! Thanks for noting that we'll have to document this. Once this feature is generally available, we might consider documenting this inline on the "Uploads" page. No need to change anything right now though.